custom_components/tahoma/climate_deh.py

```diff
@@ -36,7 +40,7 @@ def supported_features(self) -> int:
     @property
     def temperature_unit(self) -> str:
         """Return the unit of measurement used by the platform."""
-        return TEMP_CELSIUS
+        return UNIT_PERCENTAGE
```

Comment:
So weird

Reply:
I know... but the device is actually in percent, with 100 being of, and 0% being full on...
Thanks somfy for the logic